server/service/jct/api.coffee

```diff
@@ -215,6 +220,75 @@ API.service.jct.suggest.funder = (str, from, size) ->
     res.push({title: f.funder, id: f.id}) if matches
   return total: res.length, data: res
 
+API.service.jct.suggest.iac = (str, from, size) ->
+  _gather_rec = (data) ->
+    rec = {
+      'id': data.ror,
+      'title': data.title,
+      'country': data.country ? '',
+      'ror': data.ror
+    }
+    return rec
+
+  if !str
+    return total: 0, data: []
+  if !size
+    size = 10
+  str = str.toLowerCase().trim()
+
+  q = {
+    "query": {
+      "function_score" : {
+        "query" : {
+          "bool" : {
+            "should" : [
+              {"prefix" : {"index.title.exact" : str}},
+              {"prefix" : {"index.aliases.exact" : str}},
+              {"prefix" : {"index.ror.exact" : str}},
+              {"match" : {"index.title" : str}},
+              {"match" : {"index.alts" : str}},
+              {"match" : {"index.ror" : str}},
+              {"match" : {"index.acronyms.exact" : str}}
```

Comment:
This wouldn't work with `.exact` but also I have updated the importer so that acronyms go into `index.aliases` so this match query can just come out

Reply:
done